src/core/api.ts

```diff
@@ -25,9 +25,12 @@ export interface GoslingApi {
     zoomToExtent(viewId: string, duration?: number): void;
     zoomToGene(viewId: string, gene: string, padding?: number, duration?: number): void;
     suggestGene(viewId: string, keyword: string, callback: (suggestions: GeneSuggestion[]) => void): void;
+    getTracksAndViews(): VisUnitApiData[];
     getViewIds(): string[];
```

Comment:
Do we want to keep this API method? It could be confusing because `getViewIds` returns the `view.uid` created by HiGlass, not `view.id` as the name would suggest.